buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,9 +1,9 @@
 object Versions {
   const val compileSdkVersion = 30
-  const val minSdkVersion = 21
+  const val minSdkVersion = 15
```

Comment:
I think I made blend `21` min because of elevation. Maybe I can add a compat version that goes back further... We'll see 🙃